src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenUsingDeclarationTests.cs

```diff
@@ -1750,92 +1714,6 @@ After declarations
             CompileAndVerify(compilation, expectedOutput: expectedOutput);
         }
 
-        [Fact]
-        public void UsingDeclarationAsyncPatternWithTaskLikeReturnTest()
```

Comment:
Consider adjusting this test rather than removing.

Reply:
Async and ref struct can't co-exist, so we can never implement async dispose via pattern